odl/trafos/fourier.py

```diff
@@ -1645,6 +1647,211 @@ def inverse(self):
             sign=sign, tmp_r=self._tmp_r, tmp_f=self._tmp_f)
 
 
+class NonUniformFourierTransformBase(Operator):
+    """Non uniform Fast Fourier Transform.
```

Comment:
"Fast" is an implementation detail which we typically do not include in headers in ODL.

Reply:
I see. But in this case isn't it a bit more relevant since this implementation results in approximation?

Reply:
Ok, then this is ignorance on my end. We actually get different results from the fast vs "normal" nfft?

Reply:
Well up to a certain accuracy yes.
> The generalisation of the FFT is an approximative algorithm and has computational complexity O (N log N + log (1/ε) M), where ε denotes the desired accuracy